xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1469,6 +1429,35 @@ HDR_STATUS CWIN32Util::GetWindowsHDRStatus()
 
   return status;
 }
+#endif // TARGET_WINDOWS_DESKTOP
+
+HDR_STATUS CWIN32Util::GetWindowsHDRStatus()
+{
+  HDR_STATUS status = HDR_STATUS::HDR_UNKNOWN;
+
+#ifdef TARGET_WINDOWS_STORE
+
+  if (CSysInfo::GetWindowsDeviceFamily() == CSysInfo::Xbox &&
```

Comment:
It's only a personal preference but since Visual Studio editor grey out not used code path (desktop/UWP) I prefer no extra blank lines inside `#ifdef` code blocks.

i.e. this instead of current:

![after](https://github.com/xbmc/xbmc/assets/58434170/bf5f7cd8-982a-467a-bc97-fc4cf7db3015)
 
here and similar others

Reply:
I don't really care one way or the other. Updated.